src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java

```diff
@@ -198,13 +198,15 @@ private String getSubscriptionDetails(String newSubscription, String oldSubscrip
     private boolean isEventAllowedToProceedWithValidData(NotificationWrapper notificationWrapper,
                                                          NotificationEventType notificationType) {
         boolean isAllowed = true;
-        if (NotificationEventType.REQUEST_INFO_INCOMPLETE.equals(notificationType)) {
+        if (REQUEST_INFO_INCOMPLETE.equals(notificationType)) {
             if (StringUtils.isEmpty(notificationWrapper.getNewSscsCaseData().getInformationFromAppellant())
                     || "No".equalsIgnoreCase(notificationWrapper.getNewSscsCaseData().getInformationFromAppellant())) {
                 isAllowed = false;
 
                 log.error("Request Incomplete Information with empty or no Information From Appellant for ccdCaseId {}.", notificationWrapper.getNewSscsCaseData().getCcdCaseId());
             }
+        } else if (CASE_UPDATED.equals(notificationType)) {
```

Comment:
Does this stop all Case updated notifications from being sent out?

Reply:
yes, at the moment we don't have template for case updates and case updates is internal event. we don't want to notify to user

Reply:
Ok. It should also be removed from the ccd spreadsheet at some point too.